packages/flutter/lib/src/material/scrollbar_theme.dart

```diff
@@ -0,0 +1,284 @@
+      mainAxisMargin,
+      minThumbLength,
+    );
+  }
+
+  @override
+  bool operator ==(Object other) {
+    if (identical(this, other))
+      return true;
+    if (other.runtimeType != runtimeType)
+      return false;
+    return other is ScrollbarThemeData
+      && other.thickness == thickness
+      && other.showTrackOnHover == showTrackOnHover
+      && other.isAlwaysShown == isAlwaysShown
+      && other.radius == radius
+      && other.thumbColor == thumbColor
+      && other.trackColor == trackColor
+      && other.trackBorderColor == trackBorderColor
+      && other.crossAxisMargin == crossAxisMargin
+      && other.mainAxisMargin == mainAxisMargin
+      && other.minThumbLength == minThumbLength;
+  }
+
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
+    super.debugFillProperties(properties);
+    properties.add(DiagnosticsProperty<MaterialStateProperty<double?>>('thickness', thickness, defaultValue: null));
+    properties.add(DiagnosticsProperty<bool>('showTrackOnHover', showTrackOnHover, defaultValue: null));
+    properties.add(DiagnosticsProperty<bool>('isAlwaysShown', isAlwaysShown, defaultValue: null));
+    properties.add(DiagnosticsProperty<Radius>('radius', radius, defaultValue: null));
+    properties.add(DiagnosticsProperty<MaterialStateProperty<Color?>>('thumbColor', thumbColor, defaultValue: null));
+    properties.add(DiagnosticsProperty<MaterialStateProperty<Color?>>('trackColor', trackColor, defaultValue: null));
+    properties.add(DiagnosticsProperty<MaterialStateProperty<Color?>>('trackBorderColor', trackBorderColor, defaultValue: null));
+    properties.add(DiagnosticsProperty<double>('crossAxisMargin', crossAxisMargin, defaultValue: null));
+    properties.add(DiagnosticsProperty<double>('mainAxisMargin', mainAxisMargin, defaultValue: null));
+    properties.add(DiagnosticsProperty<double>('minThumbLength', minThumbLength, defaultValue: null));
+  }
+
+  static MaterialStateProperty<T>? _lerpProperties<T>(
+    MaterialStateProperty<T>? a,
+    MaterialStateProperty<T>? b,
+    double t,
+    T Function(T?, T?, double) lerpFunction,
+    ) {
+    // Avoid creating a _LerpProperties object for a common case.
+    if (a == null && b == null)
+      return null;
+    return _LerpProperties<T>(a, b, t, lerpFunction);
+  }
+}
+
+class _LerpProperties<T> implements MaterialStateProperty<T> {
+  const _LerpProperties(this.a, this.b, this.t, this.lerpFunction);
+
+  final MaterialStateProperty<T>? a;
+  final MaterialStateProperty<T>? b;
+  final double t;
+  final T Function(T?, T?, double) lerpFunction;
+
+  @override
+  T resolve(Set<MaterialState> states) {
+    final T? resolvedA = a?.resolve(states);
+    final T? resolvedB = b?.resolve(states);
+    return lerpFunction(resolvedA, resolvedB, t);
+  }
+}
+
+/// Applies a scrollbar theme to descendant [Scrollbar] widgets.
+///
+/// Descendant widgets obtain the current theme's [ScrollbarTheme] object using
```

Comment:
[ScrollbarTheme] object => [ScrollbarThemeData]

or you could just say "scrollbar theme configuration".